packages/gatsby/content/getting-started/migration.md

```diff
@@ -69,7 +62,42 @@ Run `npx @yarnpkg/doctor .` (or `yarn dlx @yarnpkg/doctor .`) in your project to
 ➤ YN0000: Failed with errors in 5.12s
 ```
 
-Note that the doctor is intended to report any potential issue - it's then up to you to decide whether they are a false positive or not (for example it won't traverse Git repositories). For this reason we don't recommend using it as a CI tool.
+In this case, the doctor noticed that `testSequencer.js` depends on a package without listing it as a proper dependency, that `webpack.config.js` was using a loader name without passing to `require.resolve` (which is unsafe, as it means the loader won't actually be loaded from `webpack-dev-server`'s dependencies), and that a test checks the content of the `node_modules` folder (which doesn't exist anymore).
+
+### Enabling it
+
+1. Look into your `.yarnrc.yml` file for the [`nodeLinker`](/configuration/yarnrc#nodeLinker) setting
+2. If you don't find it, or if it's set to `pnp`, then it's all good: you're already using Plug'n'Play!
+3. Otherwise, remove it from your configuration file
+4. Run `yarn install`
+5. Various files may have appeared; check [this article](/getting-started/qa#which-files-should-be-gitignored) to see what to put in your gitignore
+6. Commit the changes
+
+### Editor support
+
+We have a [dedicated documentation](/getting-started/editor-sdks), but if you're using VSCode (or some other IDE with Intellisense-like feature) the gist is:
+
+1. Install the [ZipFS](https://marketplace.visualstudio.com/items?itemName=arcanis.vscode-zipfs) VSCode extension
+2. Make sure that `typescript`, `eslint`, `prettier`, ... all dependencies typically used by your IDE extensions are listed at the *top level* of the project (rather than in a random workspace)
+3. Run `yarn dlx @yarnpkg/pnpify --sdk vscode`
+4. Optionally commit the changes to avoid your contributors having to follow the same procedure
```

Comment:
I don't think this should be optional, it's really annoying to have to do stuff to run peoples projects